observability/logging/cluster-logging-support.adoc

```diff
@@ -51,6 +51,11 @@ LokiStack development is ongoing. Not all APIs are currently supported.
 include::modules/cluster-logging-maintenance-support-list.adoc[leveloffset=+1]
 include::modules/unmanaged-operators.adoc[leveloffset=+1]
 
+[id="support-exception-for-coo-logging-ui-plugin"]
+== Support exception for the Logging UI Plugin
+
+Until the approaching General Availability (GA) release of the Cluster Observability Operator (COO), which is currently in link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview] (TP), Red Hat provides support to customers who are using Logging 6.0 or later with the COO for its Logging UI Plugin on {product-title} 4.16 or later. This support exception is temporary as the COO includes several independent features, some of which are still TP features, but the Logging UI Plugin is ready for GA.
```

Comment:
Inconsistent use of logging and Logging in files changed.

Reply:
Yes, it was not my choice, it is inconsistent between the docs.

Reply:
The official short name per legal & marketing that can be viewed in the official product name sheet on the red hat network is 'logging', lower case. While I have typically only used uppercase when compliance with Sentence case calls for it, I think it is appropriate here. Thank you for your work @max-cx, great job!